Docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.8.4-slim
+ENV PYTHONIOENCODING UTF-8
+ARG DEBIAN_FRONTEND=noninteractive
+
+COPY ./config /config
+
+ENV CONFIG_FILE_PATH=/config/config.py
+ENV WORK_DIR=/work
+
+RUN chmod 777 /config/install.sh /config/start.sh /config/config.sh &&\
+    pip3 install -r /config/async-requirements.txt &&\
+    pip3 install -r /config/sync-requirements.txt &&\
+    /config/install.sh
+
+WORKDIR /work
```

Comment:
I might prefer /app or /gunicorn. Is the expectation that users will volume mount this path or that they will use `FROM` and add their code to this path?

Reply:
The set path is /work, but there is no problem even if the user uses a different path. Users can enter other environment variables to suit their environment.

So I put everything related to the settings in /config so that it doesn't overlap with the user's configuration.